querydsl-sql/src/main/java/com/querydsl/sql/AbstractSQLQuery.java

```diff
@@ -622,6 +622,11 @@ protected void cleanupMDC() {
         MDC.remove(MDC_PARAMETERS);
     }
 
+    /**
+     * Set whether literals are used in SQL strings (default: false)
+     *
+     * @param useLiterals true for literals and false for bindings
```

Comment:
>    /**
     * Set whether literals are used in SQL strings **instead of parameter bindings** (default: false)
     *
     * `@param` useLiterals true for literals and false for bindings
     */

Is this maybe a good change?